### PR TITLE
Sort imports in Scala sources (except docs)

### PR DESCRIPTION
**Pull request by ihostage**
Only imports was changed in source code. I also reorganized the `.scalafmt.conf` but without changes in settings (except sorting imports of course 😄 ).

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,21 +1,63 @@
...
+  rules = [
+    AvoidInfix, # https://scalameta.org/scalafmt/docs/configuration.html#avoidinfix
+    RedundantParens, # https://scalameta.org/scalafmt/docs/configuration.html#redundantparens
+    SortModifiers, # https://scalameta.org/scalafmt/docs/configuration.html#sortmodifiers
+    PreferCurlyFors, # https://scalameta.org/scalafmt/docs/configuration.html#prefercurlyfors
+    Imports, # https://scalameta.org/scalafmt/docs/configuration.html#imports
+  ]
+  sortModifiers.order = ["private", "protected", "final", "sealed", "abstract", "implicit", "override", "lazy"]
+  imports {
+    expand = true
```
Comment (ihostage) [inline .scalafmt.conf:58]: The same as deprecated rewrite rule `ExpandImportSelectors`.

File: .scalafmt.conf [hunk 2]
```diff
@@ -1,21 +1,63 @@
 # ATTENTION:
 # Keep this in sync with documentation/.scalafmt.conf
-runner.dialect = scala212
-align.preset = true
-align.allowOverflow = true
-assumeStandardLibraryStripMargin = true
-danglingParentheses.preset = true
```
Comment (ihostage) [inline .scalafmt.conf:7]: It's default behaviour, I just removed it.

**Review by ihostage — COMMENTED** [review]
A few comments about changes in `.scalafmt.conf`

File: documentation/manual/working/commonGuide/akka/code/scalaguide/akka/typed/oo/HelloActor.scala [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 
-// #oo-hello-actor
-import akka.actor.typed.ActorRef
-import akka.actor.typed.Behavior
 import akka.actor.typed.scaladsl.AbstractBehavior
 import akka.actor.typed.scaladsl.ActorContext
 import akka.actor.typed.scaladsl.Behaviors
+// #oo-hello-actor
+import akka.actor.typed.ActorRef
+import akka.actor.typed.Behavior
```
Comment (mkurz) [inline documentation/manual/working/commonGuide/akka/code/scalaguide/akka/typed/oo/HelloActor.scala:12]: @ihostage This is a problem, because it breaks the documentation sample (here the three imports above wil not be shown in the docs anymore).
There are more cases not jsut this one.

File: documentation/manual/working/commonGuide/configuration/code/Log4j2LoggerConfigurator.scala [hunk 4]
```diff
@@ -7,24 +7,12 @@ import java.io.File
...
-
 import org.slf4j.ILoggerFactory
-
-import org.apache.logging.log4j.LogManager
-import org.apache.logging.log4j.core._
-import org.apache.logging.log4j.core.config.Configurator
-//###skip: 1
- */
-
-import play.api.Mode
```
Comment (mkurz) [inline documentation/manual/working/commonGuide/configuration/code/Log4j2LoggerConfigurator.scala:22]: @ihostage Similar problem here... scalafmt just removes the comment which is important for the docs....

File: documentation/manual/working/commonGuide/database/code/sdatabase/ScalaApplicationDatabase.scala [hunk 5]
```diff
@@ -5,10 +5,11 @@
 package sdatabase
 //#scala-jdbc-database
```
Comment (mkurz) [inline documentation/manual/working/commonGuide/database/code/sdatabase/ScalaApplicationDatabase.scala:6]: Why is there no space between `//#`? I thought scalamt will always put a space before first character of a comment(?)

File: documentation/manual/working/commonGuide/database/code/sdatabase/ScalaControllerInject.scala [hunk 6]
```diff
@@ -7,6 +7,7 @@ package sdatabase
 // #inject-controller
 // ###insert: package controllers
```
Comment (mkurz) [inline documentation/manual/working/commonGuide/database/code/sdatabase/ScalaControllerInject.scala:8]: Here for example there is a space betwen `//` and `#`... I thought I saw in the last pull requests that scalafmt will insert a space everywhere.

**Comment by mkurz** [comment]
@ihostage I added some comments. Unfortunately it's not so easy to just run scalafmt... I think we need to manually adjust some pieces of code.... (BTW: If you want to make your browser sweat and show all diffs you can use `Array.from(document.getElementsByClassName('js-diff-load')).forEach((el) => el.click());` in your browser's js console :laughing:)

File: documentation/.scalafmt.conf [hunk 7]
```diff
@@ -1,22 +1,65 @@
...
+
+# Rewrite Rules https://scalameta.org/scalafmt/docs/configuration.html#rewrite-rules
+rewrite {
+  rules = [
+    AvoidInfix, # https://scalameta.org/scalafmt/docs/configuration.html#avoidinfix
+    RedundantParens, # https://scalameta.org/scalafmt/docs/configuration.html#redundantparens
+    SortModifiers, # https://scalameta.org/scalafmt/docs/configuration.html#sortmodifiers
+    PreferCurlyFors, # https://scalameta.org/scalafmt/docs/configuration.html#prefercurlyfors
+//    TODO: enable Imports rule
+//    Imports, # https://scalameta.org/scalafmt/docs/configuration.html#imports
```
Comment (ihostage) [inline documentation/.scalafmt.conf:56]: So, disabled yet. Will be enabled in separate PR.

Comment (mkurz) [reply documentation/.scalafmt.conf:56]: Yes, let's merge this first.
